src/utils/createReducer.ts

```diff
@@ -1,5 +1,7 @@
+import { BehaviorState, MessagesState, QuickButtonsState } from '../store/types'
+
 export const createReducer = <S>(
   reducer: { [key: string]: Function },
-  state: { [key: string]: S },
+  state: BehaviorState | MessagesState | QuickButtonsState,
```

Comment:
Do you need this union?
Why don't you use `S`?

Reply:
Because I didn't noticed 😝 . Fixed!